src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -128,3 +129,23 @@ Napi::Value ModelWrap::is_dynamic(const Napi::CallbackInfo& info) {
     const auto result = _model->is_dynamic();
     return Napi::Boolean::New(env, result);
 }
+
+Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
+    if (info.Length() != 1 || !info[0].IsNumber()) {
+        reportError(info.Env(), "Invalid argument. Expected a single number.");
+        return Napi::Undefined();
+    }
+
+    auto idx = info[0].As<Napi::Number>().Int32Value();
+    auto cm_outputs = _model->outputs();  // Output<Node>
+
+    try
+    {
```

Comment:
Please, make sure that you are using openvino [clang format](https://github.com/openvinotoolkit/openvino/blob/master/src/.clang-format) to format your code